### ISSUE TITLE
Revealed Bids stuck in (pending) - Bob ver. 0.6.2 (0.6.2)

### ISSUE THREAD

**Issue by teonbg** [kyokan/bob-wallet#306]
Reveal bids - transactions got stuck in _(pending)_

Find attached screenshots. 
<img width="1142" alt="Screen Shot 2021-02-22 at 11 57 04" src="https://user-images.githubusercontent.com/47632997/108699193-3a664800-7505-11eb-86a2-8893e4d5c8da.png">

**Comment by pinheadmz** [comment]
Your reveal was confirmed: https://e.hnsfans.com/tx/28da93ca0ec9c8effdadfb59ecdf3578836ad8fca508b2a4a2397e0758d8c8f9

Looks like it was sent as a "reveal all" because theres 7 names in the transaction. Did the wallet UI ever update to reflect this?

**Comment by RobertLowe** [comment]
I had this happen to me in 0.7.1

I think I can explain the cause.

So I "Revealed Bid" while my node was still synchronizing. And waited a couple days, nothing happened.

I zapped wallet, and retried reveal (node was synced now)

I deep cleaned and rescanned, bid was finally showing as revealed.

---

Probably related, a bid I placed on another domain also wasnt placed and stuck in pending, after deep cleaning there was no transaction.

---

TL;DR; dont use bob while your node isnt fully syncronized.

**Comment by pinheadmz** [comment]
Bob won't be able to REVEAL if the node is not fully synced - the underlying hsd will reject the request because it will think the auction either hasn't even started yet, or at least has not reached the REVEAL phase yet. My guess would be you didn't have to zap or deep clean either - just wait for sync and click reveal again.

**Comment by rithvikvibhu** [comment]
This issue seems to be resolved, so closing it. Feel free to re-open if not.
